configure.ac

```diff
@@ -78,7 +78,8 @@ m4_define([libva_lt_age],
 m4_define([libdrm_version], [2.4])
 
 # Wayland minimum version number
-m4_define([wayland_api_version], [1.0.0])
+# 1.11.0 for wl_proxy_create_wrapper
+m4_define([wayland_api_version], [1.11.0])
```

Comment:
I was not talking about you specifically - should have made that clear, sorry. I'm only here fixing things that need fixing, how maintainers want to handle deps issues is not my decision :)
Speaking of CI, why was this PR marked OK by CI anyway? Does it not build the wayland stuff?